### PR TITLE
Vsb1357 nc dmd

### PR DESCRIPTION
**Pull request by vsb1357**
Hi guys
Firstly, thanks for your efforts in writing this library and making it public. 
Secondly, a few months ago I used the library to apply DMD to some molecular fMRI data. Since the signals were noisy and more conventional DMD algorithms all failed to generate something meaningful, I implemented the algorithm as per:
Dawson et al.,"Characterizing and correcting for the effect of sensor noise in 
    the dynamic mode decomposition", 2016, Section 2.3
For my particular datasets, this algorithm also did not give me something useful. Nevertheless, thought to recommend adding the code to the main library, in case somebody else might want to do the same thing. 
The code assume noise signal is available, but with a bit of tweaking it can be extended to the cases where only statistical properties of noise is known. 
Regards,
Vahid Bokharaie

### CODE REVIEW

**Comment by ndem0** [comment]
Hi @vsb1357! We are very glad for your nice comments about PyDMD and for your contribution.
In some minutues, I will check the new code and reply back if there are some minor fixes to be done.

File: ncdmd.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    :param bool exact: flag to compute either exact DMD or projected DMD.
+        Default is False.
+    :param bool opt: flag to compute optimized DMD. Default is False.
+    """
+
+    def fit(self, Xm, Noise):
+        """
+        Compute the Dynamic Modes Decomposition to the input data.
+
+        :param X: the input snapshots.
```
[the remark this answers is not in the document]
Comment (vsb1357) [reply ncdmd.py]: Thanks for your comments. I'd fix the above-mentioned issues and will come back to you on them.

**Comment by mtezzele** [comment]
Hi @vsb1357! Any news about this PR?

**Comment by vsb1357** [comment]
If you guys want to fix things such as naming conventions and trailing spaces in the file and use it, then feel free to do cause I don't have much time to spend on this. If not, then please delete it.

**Pull request closed without merging**